src/secp256k1/src/secp256k1.c

```diff
@@ -482,7 +482,7 @@ static int secp256k1_ecdsa_sign_inner(const secp256k1_context* ctx, secp256k1_sc
             break;
         }
         is_nonce_valid = secp256k1_scalar_set_b32_seckey(&non, nonce32);
-        /* The nonce is still secret here, but it being invalid is is less likely than 1:2^255. */
```

Comment:
Missed this in my previous review, apologies.

We should not update any files under `src/scp256k1` in our repo because this is part of a subtree. Please update this in the upstream at https://github.com/bitcoin-core/secp256k1 and we'll pick it up next time we update the subtree.